doc-content/enterprise-only/bpmn/bpmn-end-events-ref.adoc

```diff
@@ -8,10 +8,12 @@ A process must contain at least one end event.
 During run time, an end event finishes the process workflow. The end event can finish only the workflow that reached it, or all workflows in the process instance, depending on the end event type.
 
 .End events
-[cols="2"]
+
+[cols="50%,50%" options="header"]
+
 |===
-h| 
-h|End
+h|End event
+h|
```

Comment:
Label second column too. Even "Icon" or something. Whatever you've been doing elsewhere. Just shouldn't have no header, as a rule of thumb for tables in general.